docs/wiki/deployment/log-aggregation.md

```diff
@@ -67,7 +67,7 @@ sourcetype = osquery:warning
 
 ### Rsyslog
```

Comment:
This feels out of place. The prior sections are common tools for monitoring a file-based logfile. Syslog is a different plugin